neuromation/cli/job.py

```diff
@@ -919,8 +921,16 @@ def format_fail(job: str, reason: Exception) -> str:
         )
         click.echo(click.style(msg, dim=True))
         await _print_logs(root, job.id)
-        res = await root.client.jobs.status(job.id)
-        sys.exit(res.history.exit_code)
+        job = await root.client.jobs.status(job.id)
+        exit_code = job.history.exit_code
+    else:
+        # Even if we detached, but the job has failed to start
+        # (most common reason - no resources), the command fails
+        if job.status == JobStatus.FAILED:
+            exit_code = 1
```

Comment:
I'm curious why you use `job.history.exit_code` for an attached job but `1` for detached one?

Reply:
When detached job fails, it means it didn't even start. Hence there's no exit code. `job.history.exit_code` for such job is `None`

Reply:
Got you.

Looks like docker uses `126` exit code for the case: https://docs.docker.com/engine/reference/run/#/exit-status
It is compliant to chroot exit codes table.
Maybe we can return `126` as well?

Reply:
Or `125`, I have no strong opinion here.

Reply:
I like 125, will update now.